style.css

```diff
@@ -1 +1,175 @@
 /*  Add your styling here */
+
+body {
+  box-sizing: border-box;
+  width: 100%;
+  height: auto;
+  margin: 0 auto;
+  padding: 1, 5rem;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
```

Comment:
This will add horizontal scrolling to your page. Remove it to fix the issue.

Whe you use `grid` it works to fill the full space when you use `auto`. No need for `width` anymore.